goodreads/vite_assets/entryPoints/booksDataTable.tsx

```diff
@@ -0,0 +1,26 @@
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
+import "@mantine/core/styles.css";
+import { MantineProvider } from "@mantine/core";
+import BooksDataTable from "../components/BooksDataTable.tsx";
+import theme from "../util/theme.ts";
+
+const scriptUrlParams = new URL(import.meta.url).searchParams;
```

Comment:
Here's the glue in the entry point that parses data that's passed in from the Django template and sets up a React root to render a specific React component.

Then, inside `BooksDataTable` it's normal React code, where we can render other components without any extra stuff.